modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -222,6 +222,14 @@ export type TxRequest = {
   latest: boolean;
 };
 
+export type CreateKeychainParamsBase = {
+  userGpgKey: SerializedKeyPair<string>;
```

Comment:
This type seems a lot looser than the previous parameter type; not sure if this is intended but a bit more context behind this is helpful

Reply:
Is this only because we want to use this type for `createBitgoKeychain`?
Maybe to be base CreateKeychainParamsBase shouldn't include this property and be extended when necessary? It is always easier to deal with concrete types, this requires less assertions etc.

Reply:
@byhow are you specifically talking about `userGpgKey: SerializedKeyPair<string>;`, because that's still the same. 

About the need for creating `CreateKeychainParamsBase` - this was because we previously had types for individual fields in a specific order, which wasn't working for all create keychain methods for both eddsa/ecdsa.

Reply:
I am more talking about `CreateKeychainParamsBase` here with the new optional types. 

Also, thanks for explaining the context behind it. I am ok with loosing the types for business needs, but if we can consider @vitalir-bg 's suggestion to extend types to differentiate between different create keychain methods, that will be more helpful. 

If we specifically want to fit the parameters to apply to all create keychain methods at the same time, I am not sure if this is a good approach for different types of creating keychain methods. But if that really is the business need, if this is a breaking change, it needs to follow the commit standards to include breaking change in the commit message.

Reply:
It's not really a breaking change. If you look back at the logic of the previous create(User/Backup/Bitgo)Keychain methods-
1. `passphrase` and `originalPasscodeEncryptionCode` were always optional anyway, the only exception being createUserKeychain for ecdsa, for which I already have a concrete error msg if it's not provided. 
2.  About `bitgoKeychain` being optional, that had to be done because there is no bitgoKeychain passed in for `createBitgoKeychain` methods, everywhere else it's needed. I'll add a new type for it so we dont have to add any asserts there. 

See the latest commit for the changes.

Reply:
> If we specifically want to fit the parameters to apply to all create keychain methods at the same time
This was what was being done before. I changed that to create a base type, then extended new types for eddsa/ecdsa.